src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    private CinemaHallDao cinemaHallDao;
```

Comment:
It seems that the `CinemaHallDao` is not being injected into this service. You should use constructor injection to inject the necessary dependencies. This is important for the decoupling and testability of your service.